misc_modules/frequency_manager/src/main.cpp

```diff
@@ -190,6 +190,58 @@ class FrequencyManagerModule : public ModuleManager::Instance {
         }
         return open;
     }
+    
+    bool quickJumpDialog() {
+        bool open = true;
+        gui::mainWindow.lockWaterfallControls = true;
+        if (ImGui::IsKeyPressed(ImGuiKey_Enter) || ImGui::IsKeyPressed(ImGuiKey_Escape)) {
+            open = false;
+        }
+        std::string id = "Quick Jump##freq_manager_quick_jump_popup_" + name;
+        ImGui::OpenPopup(id.c_str());
+        ImGui::SetNextWindowSize(ImVec2(800,0));
+        if (ImGui::BeginPopupModal(id.c_str(),nullptr, ImGuiWindowFlags_NoResize)) {
+            char buf[1024];
+            strcpy(buf, quickJumpSearch.c_str());
+            if(ImGui::InputText("Search",buf, 1024)){
+                quickJumpSearch = buf;
+            }
+            // Bookmark list
+            if (ImGui::BeginTable(("freq_manager_quick_jump_bkm_table" + this->name).c_str(), 2, ImGuiTableFlags_Borders | ImGuiTableFlags_RowBg | ImGuiTableFlags_ScrollY, ImVec2(0, 200))) {
+                ImGui::TableSetupColumn("Name");
+                ImGui::TableSetupColumn("Bookmark");
+                ImGui::TableSetupScrollFreeze(2, 1);
+                ImGui::TableHeadersRow();
+                for (auto& [name, bm] : this->bookmarks) {
+                    std::string lowerJumpSearch = quickJumpSearch;
+                    for (auto& c : lowerJumpSearch){
+                        c = tolower(c);
+                    }
+                    std::string lowerName = name;
+                    for (auto& c : lowerName) {
+                        c = tolower(c);
+                    }
+                    if (lowerName.find(lowerJumpSearch) == std::string::npos) { continue; }
+                    ImGui::TableNextRow();
+                    ImGui::TableSetColumnIndex(0);
+                    ImVec2 min = ImGui::GetCursorPos();
+
+                    if (ImGui::Selectable((name + "##freq_manager_quick_jump_bkm_name" + this->name).c_str(), &bm.selected, ImGuiSelectableFlags_SpanAllColumns | ImGuiSelectableFlags_SelectOnClick)) {
+                        applyBookmark(bm, gui::waterfall.selectedVFO);
+                    }
+                    ImGui::TableSetColumnIndex(1);
+                    ImGui::Text("%s %s", utils::formatFreq(bm.frequency).c_str(), demodModeList[bm.mode]);
+                    ImVec2 max = ImGui::GetCursorPos();
+                }
+                ImGui::EndTable();
+            }
+            if (ImGui::Button("Ok",ImVec2(ImGui::GetContentRegionAvail().x,0))) {
```

Comment:
```suggestion
            if (ImGui::Button("Close",ImVec2(ImGui::GetContentRegionAvail().x,0))) {
```
OK is usually used to confirm, since this button replaces the "X" I think it is more appropriate to call it "Close" instead.